docs/guides/chip_tool_guide.md

```diff
@@ -1393,3 +1393,244 @@ $ ./chip-tool any read-by-id <cluster-ids> <attribute-ids> <destination-id> <end
+
+> **Note:** Users and credentials can only be modified by whoever has the right
+> permissions, as specified in the Access Control List.
+
+To save credentials and users, you need to complete the following steps,
+described in detail in the following sections:
+
+1. Set up a user on the device.
+1. Assign a credential for the newly created user.
+
+### Step 1: Set up a user
+
+To set up a user on a door lock device with the CHIP Tool, use the following
+command pattern:
+
+```
+$ ./chip-tool doorlock set-user <operation-type> <user-index> <user-name> <user-unique-id> <user-status> <user-type> <credential-rule> <destination-id> <endpoint-id> --timedInteractionTimeoutMs <ms_value>
+```
+
+In this command:
+
+-   _<operation-type\>_ is one of the available types of operation for the user:
+
+    -   `Add` - This operation sets a new user in the slot at _<user-index\>_.
+    -   `Clear` - This operation removes an existing user from the slot at
+        _<user-index\>_.
+    -   `Modify` - This operation modifies an existing user at the slot at
+        _<user-index\>_.
+
+-   _<user-index\>_ is the index value of the user, between `1` and the maximum
+    value you set in the `NumberOfTotalUsersSupported` attribute. Setting the
+    user index to `0` will cause an error.
+-   _<user-name\>_ is the name of the user, which can have maximum 10 bytes of
+    size. Can be set to `null`.
+-   _<user-unique-id\>_ is a 4-byte number that describes the unique user ID.
+    Can be set to `null`.
+-   _<user-status\>_ can be set to `null` or to one of the following values:
+
+    -   `0` (`Available`) - This status indicates that the given user slot is
+        available for modification on the device. This value is selected is you
+        set `null` for the variable.
+    -   `1` (`OccupiedEnabled`) - This status indicates that the given user slot
+        is used and active.
+    -   `3` (`OccupiedDisabled`) - This status indicates that the given user
+        slot is used, but disabled. Unlike `0` and `1`, supporting this status
+        is optional.
+
+-   _<credential-rule\>_ is the number of credentials that must be used to
+    unlock the door lock. This parameter can be set to `null` or to one of the
+    following values:
+
+    -   `0` (Single) - One credential type is required to unlock. This value is
+        selected is you set `null` for the variable.
+    -   `1` (Dual) - Two credential types are required to unlock.
+    -   `2` (Triple) - Three credential types are required to unlock.
+
+-   _<destination-id\>_ is the ID of the door lock device.
+-   _<endpoint-id\>_ is the ID of the endpoint on the door lock device.
+-   `--timedInteractionTimeoutMs` is the timeout duration for the interaction,
+    in milliseconds (_<ms_value>_). This should allow enough time for the
+    interaction to complete.
+
+**Examples of command:**
+
+```
+$ ./chip-tool doorlock set-user 0 1 AAA 6452 1 0 0 1 1 --timedInteractionTimeoutMs 1000
```

Comment:
This should probably describe what this command will do.  As should the next example.